packages/logger-pino/source/driver.ts

```diff
@@ -110,7 +110,16 @@ export class PinoLogger implements Contracts.Kernel.Logger {
 	}
 
 	public error(message: any): void {
-		this.#log("error", message);
+		if (typeof message === "object") {
```

Comment:
Exposed logger methods represents the log level. They all should accept message defined as string type.